public/index.html

```diff
@@ -111,11 +112,19 @@ <h2>Themes</h2>
                     <button type="button" class="themeButton" id="Default">Default</button>
                     <button type="button" class="themeButton" id="RCB40">RCB40</button>
                     <button type="button" class="themeButton" id="No2TypeB">No. 2 Type B</button>
+                    <button type="button" class="themeButton" id="SpaceGroove">Space Groove</button>
                 </div>
             </div>
             <div id="footer">
                 <span>A project by <a href="https://kenellorando.com/">Ken Ellorando</a></span>
             </div>
+
         </main>
+
     </body>
+    <div class="fullscreen-bg" id="fullscreen-bg">
```

Comment:
Is this ID actually being used anywhere? I don't see it...

Reply:
I originally had a line that would "hide" this element based on the id. It is not there anymore since I am deloading the video... so one sec...

Reply:
Fixed.